src/scheduler.cc

```diff
@@ -33,14 +39,13 @@ namespace detail {
 
 				utils::match(
 				    event,
-				    [this](const event_task_available& e) {
+				    [this, &serializer](const event_task_available& e) {
```

Comment:
Nit, but I feel like there's no reason to use anything except `[&]` for the arms of this `match`.

Reply:
Well I was simply following precedent (aa0239bbf) but sure, changed 😛

Reply:
Whoops!